### PR TITLE
Multicast configuration on readers.

### PR DESCRIPTION
**Pull request by MiguelCompany**

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/743/)

File: code/XMLTester.xml [hunk 1]
```diff
@@ -4058,7 +4058,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <profiles xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles">
 -->
-    <publisher profile_name="publisher_xml_conf_multicast_locators_profile">
+    <subscriber profile_name="publisher_xml_conf_multicast_locators_profile">
```
Comment (richiware) [inline code/XMLTester.xml:4061]:
```suggestion
    <subscriber profile_name="subscriber_xml_conf_multicast_locators_profile">
```

Comment (richiware) [reply code/XMLTester.xml:4061]: If you decided to apply this suggestion, this profile name sure is used in the testing XML source code and you will change that too.

**Review by richiware — CHANGES_REQUESTED** [review]

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/745/)

**Comment by EduPonz** [comment]
@richiprosima please test this

**Review by EduPonz — APPROVED** [review]
LGTM with green CI

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/779/)

**Pull request merged**